src/static_canvas.class.ts

```diff
@@ -748,7 +763,6 @@ import { removeFromArray } from './util/internals';
      */
     renderCanvas: function(ctx, objects) {
       var v = this.viewportTransform, path = this.clipPath;
-      this.cancelRequestedRender();
```

Comment:
This is an important fix
Before, if you would called `Canvas#toCanvasElement` it would have cancelled requested renders. Or was it intentional.
I guess now there could be some nasty race condition.